packages/sandcastle/vite.config.ci.ts

```diff
@@ -0,0 +1,29 @@
+import { defineConfig, UserConfig } from "vite";
+
+import baseConfig, { cesiumPathReplace } from "./vite.config.ts";
+
+export default defineConfig(() => {
+  const cesiumBaseUrl = `${process.env.BASE_URL}Build/CesiumUnminified`;
+
+  console.log("Building Sandcastle with base url:", cesiumBaseUrl);
```

Comment:
Is this `console.log` left here intentionally?

Reply:
I thought it was helpful to be able to see in the CI output if/when there are issues. Maybe not needed but also don't think it hurts?